ehrql/query_language.py

```diff
@@ -142,6 +152,12 @@ def configure_dummy_data(
         patients.age_on(some_date) < 80)`` would give you dummy data consisting of only men
         and women who were under the age of 80 on some particular date.
 
+        _patient_weighting_<br>
+        Defines a "weight" expression that lets you control the distribution of patients.
+        Ideally a patient row will be generated with probability proportionate to its weight,
+        although this ideal will be imperfectly realised in practice. The higher the value of
+        ``oversample`` the closer this ideal will to being realised.
```

Comment:
An example as we have for additional_population_constraints would be helpful. Will this always be a `case()`?